packages/beacon-node/src/util/blobs.ts

```diff
@@ -68,28 +82,29 @@ export function computeDataColumnSidecars(
   if (blobKzgCommitments.length === 0) {
     return [];
   }
-
+  const fork = config.getForkName(signedBlock.message.slot);
   const signedBlockHeader = signedBlockToSignedHeader(config, signedBlock);
-  const fork = config.getForkName(signedBlockHeader.message.slot);
   const kzgCommitmentsInclusionProof =
     contents.kzgCommitmentsInclusionProof ?? computeKzgCommitmentsInclusionProof(fork, signedBlock.message.body);
-  const cellsAndProofs = contents.blobs.map((blob) => ckzg.computeCellsAndKzgProofs(blob));
-  const dataColumnSidecars = Array.from({length: NUMBER_OF_COLUMNS}, (_, j) => {
-    // j'th column
-    const column = Array.from({length: contents.blobs.length}, (_, i) => cellsAndProofs[i][0][j]);
-    const kzgProofs = Array.from({length: contents.blobs.length}, (_, i) => cellsAndProofs[i][1][j]);
 
-    const dataColumnSidecar = {
-      index: j,
-      column,
-      kzgCommitments: blobKzgCommitments,
-      kzgProofs,
-      signedBlockHeader,
-      kzgCommitmentsInclusionProof,
-    } as electra.DataColumnSidecar;
+  const sidecars = Array.from({length: NUMBER_OF_COLUMNS}, () => ({
```

Comment:
what is the change here? i liked the original format where we pack the correct data instead of assigning later

Reply:
Sure thing, will refactor back to the original form.  Thought this was a bit easier to read/maintain but no biggie.

Reply:
Updated.